src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Repeater/Given_ItemsRepeater.cs

```diff
@@ -127,7 +127,7 @@ public async Task When_NestedInSVAndOutOfViewportOnInitialLoad_Then_Materialized
 		[RunsOnUIThread]
 #if __MACOS__
 		[Ignore("Currently fails on macOS, part of #9282 epic")]
-#elif __ANDROID__
+#elif __ANDROID__ || __SKIA__
 		[Ignore("Currently fails https://github.com/unoplatform/uno/issues/9080")]
 #endif
 		public async Task When_NestedIRSlowlyChangeViewport_Then_MaterializedNeededItems()
```

Comment:
1. This test passes locally for me. But fails in CI
2. I'm only changing NavigationView, but this test doesn't use NavigationView at all. So the problem shouldn't be my changes exactly, but a side effect somehow.

Reply:
Discussed with David and it's okay to disable the test.